README.md

```diff
@@ -84,6 +89,9 @@ for a SH1106:
 #include "SH1106Wire.h"
 
 SH1106Wire display(0x3c, SDA, SCL);  // ADDRESS, SDA, SCL
+// By default SD1306Wire set I2C frequency to 700000, you can use set either another frequency or skip setting the frequency by providing -1 value
```

Comment:
this should be SH1106Wire. I would use 'By default the I2C frequency of SH1106Wire is ' etc

Reply:
Done.